<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<arg name="left_is_green" default="True"/>

	<param name="reel" value="1"/>
	<param name="left_is_green" value="$(arg left_is_green)"/>


	<param name='max_hue_red' value='20'/>
	<param name='min_hue_red' value='280'/>
	<param name='max_hue_green' value='190'/>
	<param name='min_hue_green' value='90'/>
	<param name='min_sat_red' value='60'/>
	<param name='min_sat_green' value='60'/>
	<param name='max_sat_red' value='255'/>
	<param name='max_sat_green' value='255'/>
	<param name='min_val_red' value='20'/>
	<param name='min_val_green' value='20'/>
	<param name='max_val_red' value='255'/>
	<param name='max_val_green' value='255'/>
	<param name='lim_haut' value='52'/>
	<param name='lim_bas' value='100'/>
	<param name='thick' value='10'/>
	<param name='rgb' value='1'/>
	

	<!-- Run nodes -->
	<node pkg="course_2023_pkg" type="dir_indicator.py" name="dir_indicator" required="True" output="screen"/>




</launch>